packages/common/editors/EditorContainer.js

```diff
@@ -22,14 +24,13 @@ class EditorContainer extends React.Component {
     onCommit: PropTypes.func,
     onCommitCancel: PropTypes.func,
     firstEditorKeyPress: PropTypes.string,
-    width: PropTypes.number,
-    top: PropTypes.number,
-    left: PropTypes.number,
+    width: PropTypes.number.isRequired,
     scrollLeft: PropTypes.number,
-    scrollTop: PropTypes.number
+    scrollTop: PropTypes.number,
+    position: PropTypes.object.isRequired
```

Comment:
I am not that familiar with the really old syntax, should we add the shape of the position?

Reply:
Good idea, I will do that in a separate PR to keep this simple.